packages/odyssey-react-mui/src/SearchField.tsx

```diff
@@ -37,6 +37,15 @@ export type SearchFieldProps = {
    * If `true`, the component will receive focus automatically.
    */
   hasInitialFocus?: boolean;
+  /**
+   * The id of the `input` element.
+   */
+  id?: string;
+  inputRef?: Ref<HTMLInputElement>;
```

Comment:
Why are we adding an `inputRef` here?

Reply:
Can we pass stuff into `SearchField` that makes it work with the table rather than passing in the whole ref?

Even if we need a ref, we should use `useImperativeHandle` instead of passing the ref directly down.